azext_iot/_help.py

```diff
@@ -29,23 +32,26 @@
     examples:
     - name: Create an edge enabled IoT device with default authorization (shared private key).
       text: >
-        az iot hub device-identity create -n myhub -d mydevice -ee
+        az iot hub device-identity create -n [IoTHub Name] -d [Device ID] -ee
     - name: Create an IoT device with self-signed certificate authorization,
             generate a cert valid for 10 days then use its thumbprint.
       text: >
-        az iot hub device-identity create -n myhub -d mydevice -am x509_thumbprint --valid-days 10
+        az iot hub device-identity create -n [IoTHub Name] -d [Device ID] 
+        -am x509_thumbprint --valid-days 10
     - name: Create an IoT device with self-signed certificate authorization,
             generate a cert of default expiration (365 days) and output to target directory.
       text: >
-        az iot hub device-identity create -n myhub -d mydevice -am x509_thumbprint --output-dir /path/to/output
+        az iot hub device-identity create -n [IoTHub Name] -d [Device ID] -am x509_thumbprint 
```

Comment:
For understanding, what is your general strategy here, IoT Hub and Device Id's are obvious code replacement values, but you skip some other potential variables (like --output-dir and --valid-days)

Reply:
(same applies to other commands)

Reply:
They theory is that without replacement of the value the command may fail in a non-obvious way; e.g. 403, 404,  405, etc. With the "/path/to/output" you will end up with a file system access failure or completion of the request.  From what I have seen, "most" MSFT public docs follow this principle.